### ISSUE TITLE
"Read More" needs to be properly internationalized

### ISSUE THREAD

**Issue by mathetos** [impress-org/Google-Places-Reviews#7]
@mathetos commented on [Mon Jul 11 2016](https://github.com/WordImpress/Google-Places-Reviews-Pro/issues/40)

The "Read More" and "Close" strings in the Readmore.js need to be available for translation. I'm not sure the best way to implement that, except maybe to allow that to be changed manually via the widget settings.

Alternatively, we could check for the string in a localized file, and then load it in the init function. 

Bottom line is that we have un-translateable strings and need to resolve that.

**Comment by mathetos** [comment]
As reported here:
https://wordpress.org/support/topic/read-more-close-text-chance-in-other-language?replies=2#post-8627795

**Comment by DevinWalker** [comment]
Issue moved to [WordImpress/Google-Places-Reviews-Pro #57](https://github.com/WordImpress/Google-Places-Reviews-Pro/issues/57) via [**ZenHub**](https://www.zenhub.com/)
